yb-voyager/cmd/exportData.go

```diff
@@ -201,6 +201,11 @@ func exportData() bool {
 			// 3. start debezium with configration to read changes from the created replication slot, publication.
 
 			if !dataIsExported() { // if snapshot is not already done...
+				err = checkIfTablesHaveReplicationIdentityFull(finalTableList)
```

Comment:
This check can be done one block above, (line 202). Irrespective of whether snapshot is done, if it is PG and live -migration, we should perform this check

Reply:
Done